index.d.ts

```diff
@@ -0,0 +1,56 @@
+import { Readable } from 'stream'
```

Comment:
```suggestion
import { Readable, Transform } from 'stream'
```

I believe you can just import `Transform` from here.